internal/infoblox/infoblox.go

```diff
@@ -784,3 +803,22 @@ func lookupEnvAtoi(key string, fallback int) (i int) {
 	}
 	return
 }
+
+func deserializeEAs(extAttrJSON string) (map[string]interface{}, error) {
+	extAttrs := make(map[string]interface{})
+	if extAttrJSON != "" {
+		// Check if the input is stringified JSON
+		var intermediate string
+		if err := json.Unmarshal([]byte(extAttrJSON), &intermediate); err == nil {
```

Comment:
code continues even if error occures, is it ok ?

Reply:
I agree this may not be the cleanest of solutions, but I fail to come up with a neater alternative. 

If it successfully unmarshals `extAttrJSON` into a string, it means `extAttrJSON` was a stringified JSON, and should be marshalled again (hence the `extAttrJSON = intermediate` declaration).
If not, it proceeds to unmarshal `extAttrJSON` as a normal JSON object into `extAttrs`. 

Do you happen to see a better alternative on approaching this situation?